lib/builders/ProjectBuilder.js

```diff
@@ -205,7 +205,8 @@ class ProjectBuilder {
             '// GENERATED FILE - DO NOT EDIT\n' +
             'apply from: "cdv-gradle-name.gradle"\n' +
             'include ":"\n' +
-            settingsGradlePaths.join(''));
+            settingsGradlePaths.join('') +
+            '\nif (file("settings-additional.gradle").exists()) { apply from: "settings-additional.gradle" }\n');
```

Comment:
we have a concept of `build-extras.gradle` which is virtually the same thing but for the build.gradle file. So I'm thinking to keep naming consist we should actually look for `settings-extra.gradle`